internal/server/boltdbstate/app_operation_test.go

```diff
@@ -63,25 +63,31 @@ func TestAppOperation(t *testing.T) {
 		require.Equal("A", b.Id)
 		require.Equal(uint64(1), b.Sequence)
 
-		{
-			// Get it by sequence
-			raw, err = op.Get(s, &pb.Ref_Operation{
-				Target: &pb.Ref_Operation_Sequence{
-					Sequence: &pb.Ref_OperationSeq{
-						Application: b.Application,
-						Number:      b.Sequence,
-					},
+		// Get it by sequence
+		raw, err = op.Get(s, &pb.Ref_Operation{
+			Target: &pb.Ref_Operation_Sequence{
+				Sequence: &pb.Ref_OperationSeq{
+					Application: b.Application,
+					Number:      b.Sequence,
 				},
-			})
-			require.NoError(err)
-			require.NotNil(raw)
-
-			b, ok = raw.(*pb.Build)
-			require.True(ok)
-			require.NotNil(b.Application)
-			require.Equal("A", b.Id)
-			require.Equal(uint64(1), b.Sequence)
-		}
+			},
+		})
+		require.NoError(err)
+		require.NotNil(raw)
+
+		b, ok = raw.(*pb.Build)
+		require.True(ok)
+		require.NotNil(b.Application)
+		require.Equal("A", b.Id)
+		require.Equal(uint64(1), b.Sequence)
+
+		// Delete it by ID
+		err = op.Delete(s, &pb.Ref_Operation{
+			Target: &pb.Ref_Operation_Id{
+				Id: b.Id,
+			},
+		})
+		require.Nil(err)
```

Comment:
A test or two around deleting something with a non-existent operation id would be good too! Maybe in its own `Run` block.